pkg/logs/config/integration_config.go

```diff
@@ -18,6 +18,7 @@ const (
 	DockerType       = "docker"
 	JournaldType     = "journald"
 	WindowsEventType = "windows_event"
+	SnmpTrapType     = "snmp_trap"
```

Comment:
I would voucher for singular `snmp_trap` and change elsewhere to be consistent:
- snmp_traps_config=> snmp_trap_config
- snmp_traps_enabled=> snmp_trap_enabled
- expvar.NewMap("snmp_traps")=>expvar.NewMap("snmp_trap")

It's itself consistent with existing `windows_event` and `snmp_listener` that are singular.

Reply:
@AlexandreYang I would go with Jaime's suggestion here. I understand your point but in my mind we talk about "collecting _(multiple)_ trapS " in general it's more natural to just have `snmp_traps` elsewhere (as opposed to `snmp_trap` elsewhere). I don't know why `windows_event` was decided to be singular. But `snmp_listener` makes sense since there's really only one SNMP autodiscovery per Agent (although it runs multiple workers).